src/SignalR/clients/ts/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj

```diff
@@ -8,6 +8,7 @@
     <_TestSauceArgs>--verbose --no-color --configuration $(Configuration) --sauce-user "$(SauceUser)" --sauce-key "$(SauceKey)"</_TestSauceArgs>
     <_TestSauceArgs Condition="'$(BrowserTestHostName)' != ''">$(_TestSauceArgs) --use-hostname "$(BrowserTestHostName)"</_TestSauceArgs>
     <NpmTestArgs Condition="'$(DailyTests)' != 'true'">run test:local -- --no-color --configuration $(Configuration)</NpmTestArgs>
+    <SkipTests>true</SkipTests>
```

Comment:
This was a temporary hack on my temporary branch. Rebase on master and not my branch.